packages/EasyErrorHandler/src/Bridge/Symfony/Commands/AnalyzeErrorCodesCommand.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\Bridge\Symfony\Commands;
+
+use EonX\EasyErrorHandler\Bridge\Symfony\Interfaces\ErrorCodes\ErrorCodesProviderInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class AnalyzeErrorCodesCommand extends Command
+{
+    /**
+     * @var int
+     */
+    private const DEFAULT_CATEGORY_SIZE = 100;
+
+    /**
+     * @var string
+     */
+    protected static $defaultName = 'error-codes:analyze';
```

Comment:
What about to add a prefix here like doctrine does? Something like `eonx:error-codes:analyze` or `easy-error-handler:error-codes:analyze`.

Reply:
I vote for `easy-error-handler:error-codes:analyze`

Reply:
Done.